lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -1974,11 +1989,10 @@ enum Boolean {
 
 /// 2.4.8 FileBothDirectoryInformation
 /// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-fscc/270df317-9ba5-4ccb-ba00-8d22be139bc5
-/// Fields are omitted based on those omitted by FreeRDP: https://github.com/FreeRDP/FreeRDP/blob/511444a65e7aa2f537c5e531fa68157a50c1bd4d/channels/drive/client/drive_file.c#L871
 #[derive(Debug)]
 struct FileBothDirectoryInformation {
-    // next_entry_offset: u32,
-    // file_index: u32,
+    next_entry_offset: u32,
+    file_index: u32,
```

Comment:
When building `FileFullDirectoryInformation`, I decided it was more comprehensible to keep these essentially unused fields as actual fields in the structure, and add their default values [in the `new` constructor](https://github.com/gravitational/teleport/pull/12908/files#diff-983433889acb8e2c478d0a04b26bb477bd81271f29515f1ee6c56fa881e2880aR2031-R2032), rather than having them commented out and then [add the defaults in `encode`](https://github.com/gravitational/teleport/pull/12908/files#diff-983433889acb8e2c478d0a04b26bb477bd81271f29515f1ee6c56fa881e2880aL2030-L2033) like I was doing before.